docs/source/docker/getting_started/selection.rst

```diff
@@ -0,0 +1,518 @@
+            It can be either an integer or a string.
+
+
+Selection Strategy
+^^^^^^^^^^^^^^^^^^^
+
+There are several types of selection strategies, all trying to reach different objectives.
+
+.. tabs::
+
+    .. tab:: DIVERSITY
+
+        Use this strategy to **select samples such that they are as different as possible from each other**.
+
+        Can be used with **EMBEDDINGS**, **SCORES** and **numerical METADATA**. 
+        Samples with a high distance between their embeddings/scores/metadata are 
+        considered to be more *different* from each other than samples with a 
+        low distance. The strategy is specified like this:
+
+        .. code-block:: python
+
+            "strategy": {
+                "type": "DIVERSITY"
+            }
+
+        If you want to preserve a minimum distance between chosen samples, you 
+        can specify it as an additional stopping condition. The selection process
+        will stop as soon as one of the stopping criteria has been reached.
+
+        .. code-block:: python
+            :emphasize-lines: 3
+
+            "strategy": {
+                "type": "DIVERSITY",
+                "stopping_condition_minimum_distance": 0.2
+            }
+
+        Setting :code:`"stopping_condition_minimum_distance": 0.2` will remove all samples which are
+        closer to each other than 0.2. 
+        This allows you to specify the minimum allowed distance between two images in the output dataset.
+        If you use embeddings as input, this value should be between 0 and 2.0, as the embeddings are normalized to unit length.
+        This is often a convenient method when working with different data sources and trying to combine them in a balanced way.
+        If you want to use this stopping condition to stop the selection early,
+        make sure that you allow selecting enough samples by setting :code:`n_samples` or :code:`proportion_samples` high enough.
+
+        .. note:: Higher minimum distance in the embedding space results in more
+                  diverse images being selected. Furthermore, increasing the
+                  minimum distance will result in fewer samples being selected.
+
+    .. tab:: WEIGHTS
+
+        The objective of this strategy is to **select samples that have a high numerical value**.
+        
+        Can be used with **SCORES** and **numerical METADATA**. It can be specified with:
+
+        .. code-block:: python
+
+            "strategy": {
+                "type": "WEIGHTS"
+            }
+
+    .. tab:: THRESHOLD
+
+        The objective of this strategy is to only **select samples that have a numerical value fulfilling a threshold criterion**.
+        E.g. they should be bigger or smaller than a certain value.
+
+        Can be used with **SCORES** and **numerical METADATA**. It is specified as follows:
+
+        .. code-block:: python
```

Comment:
for me it's still not clear what's kept and what's removed here

Reply:
I agree, it is explained better in the usage example. I added an extra sentence:
`This will keep all samples whose value (specified by the input) is >= 20 and remove all others.`